README.md

```diff
@@ -1 +1,12 @@
-# hackmd-test
\ No newline at end of file
+# hackmd-test
+
+This is a new document written in markdown.
+
+## Results
+
+What do we think here? 
+Awesome stuff. 
+What happens when you have a paragraph? 
+Lot's of text here, we scousadl kj asdlkj asdlkj asdlj ;alskjd ;oiuwqe;oiu ;djahf lkajh cnlkjnxlkasjdh liauweh liu hfalkjsnc ;alksjd;alksj;oaeijr;oif;sakchjlkjas hclkja hekljha lkjedh liaushldiuahs dkajsh dlkajs hd. 
+This is arguably lots of gibberish.
+Is it really worth debating whether to write sentences on separate lines? This could be a good idea, but could also be annoying.
```

Comment:
```suggestion
Is it really worth debating whether to write sentences on separate lines? 
This could be a good idea, but could also be annoying.
```

Please put each sentence on a separate line!